web/src/lib/muiTheme.ts

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
```

Comment:
This has been added because typescript is letting me know that MuiTabPanel doesn't exist, but it actually does.